doc/manpages/man1/afp_lantest.1.md

```diff
@@ -4,14 +4,13 @@ afp_lantest — AFP LAN performance and directory cache testing tool
 
 # Synopsis
 
-**afp_lantest** [-34567GgVvbC] [-h *host*] [-p *port*] [-s *volume*] [-u *user*] [-w *password*]
+**afp_lantest** [-1234567bcGgKVv] [-h *host*] [-p *port*] [-s *volume*] [-u *user*] [-w *password*]
```

Comment:
The lantest doesn't support AFP 2.1/2.2 so we shouldn't add -1 and -2 to the optarg parameters here.